<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1242</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="AEDDisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>881</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>0</y>
      <width>341</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 221, 218);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="GUIConsole">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>230</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid #555;
border-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="heartmonitorBox">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>10</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid #555;
border-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>10</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid #555;
border-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;DejaVu Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Heart Rate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>50</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;DejaVu Sans&quot;;</string>
    </property>
    <property name="text">
     <string>90</string>
    </property>
   </widget>
   <widget class="QPushButton" name="powerOn">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>110</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Power On</string>
    </property>
   </widget>
   <widget class="QPushButton" name="replaceBattery">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>110</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Replace Battery</string>
    </property>
   </widget>
   <widget class="QComboBox" name="electrodePadOption">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>190</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Adult Pads</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Child Pads</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="shockDelivery">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>480</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Shock Delivery</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CPR">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>480</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>CPR</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>450</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #333;
border: 2px solid #555;
border-radius: 20px;
border-style: outset;
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #888
);
padding: 5px;
</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>60</y>
      <width>90</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid #555;
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid #555;
background-color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="batteryProgressBar">
    <property name="geometry">
     <rect>
      <x>695</x>
      <y>25</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(87, 227, 137);
background-color: rgb(0, 0, 0);
</string>
    </property>
    <property name="maximum">
     <number>102</number>
    </property>
    <property name="value">
     <number>23</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_4">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>41</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid #555;
background-color: rgb(134, 94, 60);
border-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_5">
    <property name="geometry">
     <rect>
      <x>828</x>
      <y>30</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid #555;
background-color: rgb(192, 191, 188);
border-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>311</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(245, 194, 17);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="pad2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>71</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="heartMonitor">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>30</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>HeartMonitor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="placePad">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>350</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Place pad</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="pad1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="pad4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>280</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="BoobLeft">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 165, 10);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="BoobRight">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 165, 10);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ab1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 165, 10);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ab2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 165, 10);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ab3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 165, 10);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ab4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>260</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 165, 10);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ab5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>320</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 165, 10);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ab6">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>320</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 165, 10);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_15">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>71</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(245, 194, 17);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_16">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>71</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(245, 194, 17);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_17">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(245, 194, 17);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_18">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(245, 194, 17);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="placePadIncorrectly">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Place pad Incorrectly</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="pad3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>290</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="pad1_1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="pad1_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>95</y>
      <width>41</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="pad1_3">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>220</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="pad1_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>370</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="pad1_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>295</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="padChild1">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>155</y>
      <width>121</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(94, 92, 100);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="padChild2">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>165</y>
      <width>101</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(192, 191, 188);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="padChild3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <zorder>AEDDisplay</zorder>
   <zorder>textEdit</zorder>
   <zorder>GUIConsole</zorder>
   <zorder>heartmonitorBox</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>powerOn</zorder>
   <zorder>replaceBattery</zorder>
   <zorder>electrodePadOption</zorder>
   <zorder>shockDelivery</zorder>
   <zorder>CPR</zorder>
   <zorder>textEdit_3</zorder>
   <zorder>line</zorder>
   <zorder>textEdit_2</zorder>
   <zorder>batteryProgressBar</zorder>
   <zorder>textEdit_4</zorder>
   <zorder>textEdit_5</zorder>
   <zorder>textEdit_6</zorder>
   <zorder>heartMonitor</zorder>
   <zorder>placePad</zorder>
   <zorder>BoobLeft</zorder>
   <zorder>BoobRight</zorder>
   <zorder>textEdit_15</zorder>
   <zorder>textEdit_16</zorder>
   <zorder>textEdit_17</zorder>
   <zorder>textEdit_18</zorder>
   <zorder>ab1</zorder>
   <zorder>ab2</zorder>
   <zorder>ab3</zorder>
   <zorder>ab4</zorder>
   <zorder>ab5</zorder>
   <zorder>ab6</zorder>
   <zorder>placePadIncorrectly</zorder>
   <zorder>pad1_1</zorder>
   <zorder>pad1_2</zorder>
   <zorder>pad1</zorder>
   <zorder>pad1_3</zorder>
   <zorder>pad2</zorder>
   <zorder>pad1_5</zorder>
   <zorder>pad1_4</zorder>
   <zorder>pad3</zorder>
   <zorder>pad4</zorder>
   <zorder>padChild1</zorder>
   <zorder>padChild2</zorder>
   <zorder>padChild3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1242</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
